specification/cognitiveservices/OpenAI.Inference/models/completions/streaming/chat_completions.tsp

```diff
@@ -0,0 +1,126 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+
+import "../azure_chat_extensions.tsp";
+import "../common.tsp";
+import "../functions.tsp";
+import "../chat_completions.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace Azure.OpenAI;
+
+@doc("A single, role-attributed message within a chat completion interaction.")
+model ChatMessageStreamChunk {
```

Comment:
For clarity, I'd recommend using the word "Delta" instead of "Message" in the name; they map to the same concepts, but given "delta" replaces "message" in the actual wire payload, it's likely best to stick to "descriptive" instead of "explanatory".

For comparison, [OpenAI's YAML file](https://github.com/openai/openai-openapi/blob/e145786e70bf5fc1bc73c7cd19884f445d52c383/openapi.yaml#L2832) calls this `ChatCompletionStreamResponseDelta`. It might not be a bad idea to reuse those names where we can.

Reply:
sounds good! I will go for now with `ChatCompletionsStreamDelta` and we can further tune it if you want :)